rpc/json.go

```
@@ -82,7 +95,10 @@ func (msg *jsonrpcMessage) isUnsubscribe() bool {
 }
 
 func (msg *jsonrpcMessage) namespace() string {
-	elem := strings.SplitN(msg.Method, serviceMethodSeparator, 2)
+	elem, _ := elementizeMethodName(msg.Method)
+	if len(elem) == 0 {
```

Don't ignore the error. Use it instead of `len(elem) == 0`.